libkbfs/block_journal.go

```diff
@@ -118,33 +118,6 @@ func (e blockJournalEntry) getSingleContext() (
 		"getSingleContext() erroneously called on op %s", e.Op)
 }
 
-// classifyIntoBPS either puts the given block op entry into the given
-// "Put" blockPutState or the "addRef" blockPutState, depending on the
-// op type, or it returns false.
-func (e blockJournalEntry) classifyIntoBPS(buf []byte,
```

Comment:
inlined this and `getEntryToFlush` into `getNextEntriesToFlush`, as it turns out to be clearer, I think